frontend/tailwind.config.js

```diff
@@ -0,0 +1,43 @@
+/** @type {import('tailwindcss').Config} */
+const colors = require('tailwindcss/colors');
+
+export default {
+  content: [
+    "./src/**/*.{js,jsx,ts,tsx}",
+  ],
+  theme: {
+    extend: {},
+    screens: {
+      'xs': '0px',
+      'sm': '576px',
+      'md': '768px',
+      'lg': '992px',
+      'xl': '1200px',
+      'xxl': '1400px',
+    },
+    fontFamily: {
+      'poppins-thin': ['Poppins Thin', 'sans-serif'],
```

Comment:
I think we should remove the font name from the class name. As it is, changing the font family would be a pain.